package.json

```diff
@@ -5,14 +5,14 @@
   "main": "lib/index.js",
   "scripts": {
     "prepublish": "not-in-publish || safe-publish-latest && npm run build",
-    "build": "rimraf lib && babel src -d lib",
+    "build": "rimraf lib && babel --source-maps inline src -d lib",
     "lint": "eslint src test",
     "pretest": "npm run --silent lint",
     "test": "npm run build && npm run test:coverage && npm run --silent cover:check",
     "tests-only": "npm run build && npm run test:quick",
-    "test:coverage": "babel-node node_modules/.bin/istanbul cover --report html node_modules/.bin/_mocha -- -R tap test/init.js test/*-test.js",
-    "cover:check": "istanbul check-coverage && echo code coverage thresholds met, achievement unlocked!",
-    "test:quick": "babel-node node_modules/.bin/_mocha -R tap test/init.js test/*-test.js"
+    "test:coverage": "nyc --silent node_modules/.bin/mocha -R tap test/init.js test/*-test.js",
+    "cover:check": "nyc report && echo code coverage thresholds met, achievement unlocked!",
+    "test:quick": "npm run test:coverage"
```

Comment:
the point of the "quick" script is to run *without* coverage :-) can this line be reverted?